src/quo2/components/common/not_implemented.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo2.components.common.not-implemented
+  (:require [react-native.core :as rn]))
+
+(defn not-implemented
+  [content]
+  (when content
+    [rn/view {:border-color :red :border-width 1}
```

Comment:
Style should be in `:style` prop

Reply:
@mmilad75, it is an approach within our codebase, that is documented in [guidelines](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#always-add-styles-inside-the-style-key)

Reply:
Done